### DISCUSSION TITLE
remote systems best practice for remote endpoint forensic

### DISCUSSION THREAD

**Discussion by vanh20** [Security-Onion-Solutions/securityonion#5410] [Unsupported Versions] [answered]
Hello Everyone,

   I've been looking for a recommendation as to which remote system endpoint software to [download](https://docs.securityonion.net/en/2.3/host.html) in order to troubleshoot my network of windows and linux machines from a suspected network trojan infection. My lack of skill on this matter unables me to pin the source (program/malware) from the said environments. Do I need to install ALL the endpoint softwares (Winlog/filebeat, wazuh, osquery, or even syslog and autoruns) OR which ones are the most important? This may be a newbie question to you guys, but I have to risk ridicule than not employ the full potential of Security Onion as a hunters tool. Perhaps there's a documentation I failed to read?

   Thank you guys for your kindness and forbearance.

**Comment by dougburks** [comment, marked as answer]
I've updated the [Host Visibility page of the documentation](https://docs.securityonion.net/en/2.3/host.html) as follows:

> You can send logs to Security Onion via your choice of either osquery, Beats, Wazuh, or Syslog.

> For Windows endpoints, you can optionally augment the standard Windows logging with Sysmon and/or Autoruns. Those additional logs can then be transported by whatever mechanism you chose above.

**Reply by vanh20** [reply]
Brilliant! Thanks dougburks! :)

**Reply by vanh20** [reply]
Can I ask a folllow up question? Is there any problem in using osquery, beats and Wazuh simultaneously (or any of the two) as an endpoint log collectors? Does this make any sense or will I just face multiple problems if I do so? Thanks again dougburks! :)

**Reply by vanh20** [reply]
OK... it's one or the [other](https://github.com/Security-Onion-Solutions/securityonion/discussions/4687#discussioncomment-971948) for Wazuh and Osquery. Is it the same with Syslog? Anyway, I think I got it now, more or less. Thanks! :)
